docs/blog/20230501-Daeraxa-MayUpdate.md

```diff
@@ -0,0 +1,168 @@
+
+[@savetheclocktower] has been working on some improvements for this. For example, Tree-sitter parsers are quite good at this sort of code analysis, and can identify the important parts of a source code file with [a simple query file](https://tree-sitter.github.io/tree-sitter/code-navigation-systems).
+
+[Language servers](https://microsoft.github.io/language-server-protocol/) are another potential source of symbol information. Some language servers are even capable of supplying project-wide symbol information; this would improve other `symbol-view` responsibilities, like the “Go to Declaration” command.
+
+So to make steps towards achieving this we would need to work on `symbols-view` to be more generic and then update Pulsar and its packages to take advantage of the changes.
+
+This has the potential to really upgrade the functionality and make navigating files and projects far more intuitive. It isn't ready just yet but don't worry, we will have more information to share on this feature in the coming weeks.
+
+## CI Testing speed upgrade
+
+<img src="./assets/ghtests.png">
+
+[@DeedeeG] has [made an update](https://github.com/pulsar-edit/pulsar/pull/492) to our continuous integration configuration by caching and restoring the dependencies which ends up saving us a huge amount of time, possibly over an hour each run! (Pulsar has to build and test an awful lot of packages on each run so it is a long process). This should hopefully make life just that bit easier on the development side of things.
+
+## Backend webhook
+
+<img src="./assets/package-webhook.png">
+
+[@confused-techie] has been busy adding yet another [new feature](https://github.com/pulsar-edit/package-backend/pull/156) to our package backend, this time it is a new webhook that allows us to publish messages and notifications when package authors publish and update their packages. You can check this out in action in our [backend-notifications](https://discordapp.com/channels/992103415163396136/1045131651748999239) channel on Discord.
+
+## Package website readme links overhaul
+
+<img src="./assets/package-frontend.png">
+
+Now some [changes to our package website](https://github.com/pulsar-edit/package-frontend/pull/96). There are a few separate, but related, changes by [@confused-techie] here that involve improving our package website to overhaul some of the issues we have with readme links.
+
+The first change adds shorthand string author assignment in the `package.json` file of a package. The [npm package specification(https://docs.npmjs.com/cli/v9/configuring-npm/package-json#people-fields-author-contributors)] allows the three author properties to be combined into a single string e.g.
```

Comment:
```suggestion
The first change adds shorthand string author assignment in the `package.json` file of a package. The [npm package specification](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#people-fields-author-contributors) allows the three author properties to be combined into a single string e.g.
```

Just a small syntax fix here, but otherwise to your previous comment, this section now looks perfect!